third_party/silabs/efr32_sdk.gni

```diff
@@ -91,6 +91,15 @@ declare_args() {
 
   # Enable TestEventTrigger in GeneralDiagnostics cluster
   sl_enable_test_event_trigger = false
+
+  # Temperature Sensor support
+  sl_enable_si70xx_sensor = false
```

Comment:
Could we add a comment about which board support this sensor and/or have a check depending on the board used to enable it? Unless it is available on all our matter devboard off course.

Reply:
I was planning instead of adding that information to the documentation of the sensor app instead of here. 
I can add an assert of the boards that can't support it.

Reply:
+1 for the assert